### PR TITLE
Input validation for viewIncome and minor bug fixes

### PR DESCRIPTION
**Pull request by loyhongshenggg**
Input validation for viewIncome:

Any input for years <1900 or >2200 will be treated as invalid input.

image

Bug fixes for the following:
- fxml version type error
- fxml "rainbow" color error

### CODE REVIEW

File: src/main/java/longtimenosee/logic/commands/ViewIncomeCommand.java [hunk 1]
```diff
@@ -12,7 +12,7 @@ public class ViewIncomeCommand extends Command {
 
     public static final String COMMAND_WORD = "viewIncome"; //should I change to refresh?
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + "<YEAR>" + ": chooses the year for viewing income";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " <year>" + ": chooses the year for viewing income";
     private final int targetYear;
 
     public ViewIncomeCommand(int targetYear) {
```
Comment (ReubenChay) [inline src/main/java/longtimenosee/logic/commands/ViewIncomeCommand.java:18]: i noticed in your execute method in ViewIncomeCommand that you used a variable "i", maybe you could use a more descriptive term such as income?

**Review by ReubenChay — COMMENTED** [review]
apart from the comment i left, LGTM! Thank you for your efforts!

File: src/main/java/longtimenosee/logic/commands/ViewIncomeCommand.java (hunk 1 above)
Comment (loyhongshenggg) [reply src/main/java/longtimenosee/logic/commands/ViewIncomeCommand.java:18]: Rectified! Thank you!

**Comment by loyhongshenggg** [comment]
Latest commit has the following changes:

Code
1. Added year input validation for viewIncome
2. Minor bug fixes for viewIncome calculation
3. Resized UI for bigger result box

UG
1. Collapsible menu for command summary and FAQ
2. Section for viewIncome and pin/viewPin

File: src/main/java/longtimenosee/ui/HelpWindow.java [hunk 2]
```diff
@@ -15,7 +15,8 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL =
+            "https://github.com/AY2223S1-CS2103T-W13-2/tp/blob/master/docs/UserGuide.md";
```
Comment (ReubenChay) [inline src/main/java/longtimenosee/ui/HelpWindow.java:19]: thanks for changing this

File: docs/UserGuide.md [hunk 3]
```diff
@@ -278,42 +263,42 @@ Examples:
...
+### Tracking your Income  : `viewIncome`
 
-### Tracking your Income 
-
-
-
-## Event Features 
+This allows you to sort your clients in the address book based on a specified metric.
 
+Format: `viewIncome <year>`
```
Comment (ReubenChay) [inline docs/UserGuide.md:276]: i think this might be a good place to add a UI example so we have more diagrams in our UG?

**Review by ReubenChay — COMMENTED** [review]
LGTM! I think can add UI diagram in the viewIncome UG so our UG has more diagrams

**Pull request merged**